pkg/controller/namespace/deletion/namespaced_resources_deleter.go

```diff
@@ -304,12 +306,13 @@ func (d *namespacedResourcesDeleter) finalizeNamespace(namespace *v1.Namespace)
 // deleteCollection is a helper function that will delete the collection of resources
 // it returns true if the operation was supported on the server.
 // it returns an error if the operation was supported on the server but was unable to complete.
-func (d *namespacedResourcesDeleter) deleteCollection(gvr schema.GroupVersionResource, namespace string) (bool, error) {
-	klog.V(5).Infof("namespace controller - deleteCollection - namespace: %s, gvr: %v", namespace, gvr)
+func (d *namespacedResourcesDeleter) deleteCollection(ctx context.Context, gvr schema.GroupVersionResource, namespace string) (bool, error) {
+	logger := klog.FromContext(ctx)
+	logger.V(5).Info("Namespace controller - deleteCollection", "namespace", namespace, "resource", gvr)
 
 	key := operationKey{operation: operationDeleteCollection, gvr: gvr}
 	if !d.opCache.isSupported(key) {
-		klog.V(5).Infof("namespace controller - deleteCollection ignored since not supported - namespace: %s, gvr: %v", namespace, gvr)
+		logger.V(5).Info("Namespace controller - deleteCollection ignored since not supported", "namespace", namespace, "resource", gvr)
```

Comment:
```suggestion
		logger.V(5).Info("Namespace controller - deleteCollection ignored since not supported -", "namespace", namespace, "groupVersionResource", gvr)
```

Reply:
why do we need - here?

Reply:
To have the same consistent log as the old ones. This applies to all other comments as well.

Reply:
I don't think it needs to be consistent with the old logs. After the log is structured, it is already in key value format.